sim/paladin/item_sets_pve.go

```diff
@@ -215,24 +216,14 @@ var ItemSetRadiantJudgement = core.NewItemSet(core.ItemSet{
 				Label: "S03 - Item - T2 - Paladin - Retribution 4P Bonus",
```

Comment:
Would you mind updating the comment for this above here?

Reply:
updated